generate-routes.ts

```diff
@@ -54,6 +57,7 @@ const routePaths = [
   '/networks',
   '/noise_sensors',
   '/noise_sensors/noise_thresholds',
+  '/noise_sensors/simulate',
```

Comment:
We need to add this under `routePathSubresources`

Reply:
https://github.com/seamapi/javascript-http/pull/87/commits/98b8d5de0ff2d05ebda402bd9c7c8a920bd50933